packages/web-react/src/components/ControlButton/README.md

```diff
@@ -0,0 +1,145 @@
+# ControlButton
+
+ControlButton is a simple button designed for secondary actions that adapts to its background color context.
```

Comment:
Please, do not use "secondary" or "primary" or other types that are meant to be used with the call to action "Button".